introduction.md

```diff
@@ -0,0 +1,3 @@
+There's a saying: When the student is ready the master will come...
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.